src/plugins/subsystems/drm/drm.sh

```
@@ -255,8 +259,8 @@ function get_available_connectors()
       ;;
     3) # REMOTE TARGET
       local find_conn_cmd="find $SYSFS_CLASS_DRM -name 'card*'"
```

Why not move this local to the top of the function as well?

---

@rodrigosiqueira  because there is no return value to be masked, I only touched the declarations that could mask errors

---

@rodrigosiqueira should I split them too?

---

> @rodrigosiqueira should I split them too?

By "split them" do you mean the line that I commented? If so, I think it does not hurt to split it.